man/systemd.exec.xml

```diff
@@ -2555,6 +2555,16 @@ RestrictNamespaces=~cgroup net</programlisting>
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PrivateBPF=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If set, mount a private instance of the BPF filesystem
+        on <filename>/sys/fs/bpf</filename>. Otherwise, if <varname>ProtectKernelTunables=</varname> is set,
```

Comment:
plese suffix paths that refer to dirs with `/`, in accodance to our coding style, i.e. spell out `/sys/fs/bpf/` here.